compatibility/history-proxy/__tests__/params.test.js

```diff
@@ -0,0 +1,183 @@
+const { createDataToBePassed } = require('../src/proxy');
```

Comment:
Try following  the folder patterns from others services in this repo
You can using /tests/unit ou /test/integration...